moveit_kinematics/test/test_kinematics_plugin.cpp

```diff
@@ -345,14 +345,14 @@ TEST_F(KinematicsTest, randomWalkIK)
   robot_trajectory::RobotTrajectory traj(robot_model_, jmg_);
 
   unsigned int failures = 0;
-  constexpr double NEAR_JOINT = 0.1;
```

Comment:
Change to `static` please and keep uppercase.

Reply:
Changed as suggested.